che-plugin-openshift-ext-client/src/main/resources/org/eclipse/che/ide/ext/openshift/client/OpenshiftLocalizationConstant.properties

```diff
@@ -102,8 +102,10 @@ deploy.project.window.variables.add.tooltip=Add new variable
 deploy.project.window.labels.add.tooltip=Add new label
 deploy.project.success=Project <b>{0}</b> successfully deployed to OpenShift
 
-invalid.openshift.project.name.error=Name must be a valid DNS label
-invalid.che.project.name.error=Name must be a valid Che project name
+invalid.openshift.project.name.error=Name must be a valid project name
```

Comment:
Valid DNS label is not really a good definition, name must not also start with number and be at least 3 characters long. So, either we can display this, or full definition/rexegp of project name.
Same to application name